osidb/models.py

```diff
@@ -2723,6 +2726,30 @@ def is_triage(self):
         # it is to be determined and implemented
         return False
 
+    @property
+    def is_compliance_priority(self) -> bool:
+        """
+        Check and return whether this affect module, component and update stream combination is
+        compliance priority which is defined in PS constance repo in compliance_priority.yml.
+        """
+        affect = self.affects.first()
+        if not affect:
+            return False
+
+        ps_module = PsModule.objects.filter(name=affect.ps_module).first()
```

Comment:
I am not sure about the necessity of this DB query. The PS module seems to be only used to get its name and the name is already present as part of the affect data. I actually think that this should cause troubles instead if we call at on some old/bogus affect with non-existing PS module as next `ps_module.name` would throw an `AttributeError`.

Reply:
You are right. It's a useless and slightly wrong query. Fixed.